### ISSUE TITLE
LICENSE file missing from sdist on PyPI

### ISSUE THREAD

**Issue by nehaljwani** [neithere/argh#129]

**Comment by hugovk** [comment]
This should be fixed for the next release. 

Using latest setuptools 56.0.0, it includes both `COPYING` and `COPYING.LESSER` in the sdist:

```console
$ ⌂273% [hugo:/private/tmp/argh] master ± python setup.py sdist
/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/setuptools/dist.py:463: UserWarning: Normalizing '0.26.3-dev' to '0.26.3.dev0'
  warnings.warn(tmpl.format(**locals()))
running sdist
running egg_info
creating argh.egg-info
writing argh.egg-info/PKG-INFO
writing dependency_links to argh.egg-info/dependency_links.txt
writing top-level names to argh.egg-info/top_level.txt
writing manifest file 'argh.egg-info/SOURCES.txt'
adding license file 'COPYING.LESSER' (matched pattern 'COPYING*')
adding license file 'COPYING' (matched pattern 'COPYING*')
adding license file 'AUTHORS' (matched pattern 'AUTHORS*')
reading manifest template 'MANIFEST.in'
writing manifest file 'argh.egg-info/SOURCES.txt'
running check
creating argh-0.26.3.dev0
creating argh-0.26.3.dev0/argh
creating argh-0.26.3.dev0/argh.egg-info
creating argh-0.26.3.dev0/docs
creating argh-0.26.3.dev0/test
copying files to argh-0.26.3.dev0...
copying .gitignore -> argh-0.26.3.dev0
copying .hgignore -> argh-0.26.3.dev0
copying .hgtags -> argh-0.26.3.dev0
copying .travis.yml -> argh-0.26.3.dev0
copying AUTHORS -> argh-0.26.3.dev0
copying CHANGES -> argh-0.26.3.dev0
copying COPYING -> argh-0.26.3.dev0
copying COPYING.LESSER -> argh-0.26.3.dev0
copying MANIFEST.in -> argh-0.26.3.dev0
copying Makefile -> argh-0.26.3.dev0
copying README.rst -> argh-0.26.3.dev0
copying coverage.sh -> argh-0.26.3.dev0
copying reqs-dev.txt -> argh-0.26.3.dev0
copying runtestloop.sh -> argh-0.26.3.dev0
copying setup.cfg -> argh-0.26.3.dev0
copying setup.py -> argh-0.26.3.dev0
copying tox.ini -> argh-0.26.3.dev0
copying argh/__init__.py -> argh-0.26.3.dev0/argh
copying argh/assembling.py -> argh-0.26.3.dev0/argh
copying argh/compat.py -> argh-0.26.3.dev0/argh
copying argh/completion.py -> argh-0.26.3.dev0/argh
copying argh/constants.py -> argh-0.26.3.dev0/argh
copying argh/decorators.py -> argh-0.26.3.dev0/argh
copying argh/dispatching.py -> argh-0.26.3.dev0/argh
copying argh/exceptions.py -> argh-0.26.3.dev0/argh
copying argh/helpers.py -> argh-0.26.3.dev0/argh
copying argh/interaction.py -> argh-0.26.3.dev0/argh
copying argh/io.py -> argh-0.26.3.dev0/argh
copying argh/utils.py -> argh-0.26.3.dev0/argh
copying argh.egg-info/PKG-INFO -> argh-0.26.3.dev0/argh.egg-info
copying argh.egg-info/SOURCES.txt -> argh-0.26.3.dev0/argh.egg-info
copying argh.egg-info/dependency_links.txt -> argh-0.26.3.dev0/argh.egg-info
copying argh.egg-info/top_level.txt -> argh-0.26.3.dev0/argh.egg-info
copying docs/Makefile -> argh-0.26.3.dev0/docs
copying docs/changes.rst -> argh-0.26.3.dev0/docs
copying docs/conf.py -> argh-0.26.3.dev0/docs
copying docs/contributors.rst -> argh-0.26.3.dev0/docs
copying docs/cookbook.rst -> argh-0.26.3.dev0/docs
copying docs/index.rst -> argh-0.26.3.dev0/docs
copying docs/projects.rst -> argh-0.26.3.dev0/docs
copying docs/reference.rst -> argh-0.26.3.dev0/docs
copying docs/similar.rst -> argh-0.26.3.dev0/docs
copying docs/subparsers.rst -> argh-0.26.3.dev0/docs
copying docs/tutorial.rst -> argh-0.26.3.dev0/docs
copying test/__init__.py -> argh-0.26.3.dev0/test
copying test/base.py -> argh-0.26.3.dev0/test
copying test/test_assembling.py -> argh-0.26.3.dev0/test
copying test/test_decorators.py -> argh-0.26.3.dev0/test
copying test/test_dispatching.py -> argh-0.26.3.dev0/test
copying test/test_integration.py -> argh-0.26.3.dev0/test
copying test/test_interaction.py -> argh-0.26.3.dev0/test
copying test/test_regressions.py -> argh-0.26.3.dev0/test
copying test/test_utils.py -> argh-0.26.3.dev0/test
Writing argh-0.26.3.dev0/setup.cfg
creating dist
Creating tar archive
removing 'argh-0.26.3.dev0' (and everything under it)
```

I confirmed they're in the generated `.tar.gz` as well.

Can this issue be closed?

**Comment by nehaljwani** [comment]
I would close after verifying the uploaded archive on pypi :-)

**Comment by neithere** [comment]
I believe this should be fixed in the latest release (0.27).
